issue.js

```diff
@@ -12,6 +14,15 @@ function Issue(number) {
       return priorityLabel && getNumber(priorityLabel);
    };
 
+   this.getPriorityLabel = function() {
+      return priorityLabel && priorityLabel.title;
+   };
+
+   this.getPreviousPriority = function() {
+      return previousPriorityLabel && getNumber(previousPriorityLabel)
+       || this.getPriority();;
```

Comment:
This expression is kinda hard to follow. This is basically saying that if `previousPriorityLabel` is truthy, then return `getNumber(previousPriorityLabel)`, otherwise return `this.getPriority()` right? Maybe some parens would make this clearer.

Also there's a double semicolon.

Reply:
Yes, that's correct. Sure. parens for clarity are fine.